frontend/src/concepts/pipelines/content/tables/experiment/renderUtils.tsx

```diff
@@ -12,6 +12,11 @@ export const ExperimentCreated: ExperimentUtil = ({ experiment }) => {
   return <PipelinesTableRowTime date={createdDate} />;
 };
 
+export const LastExperimentRunsStarted: ExperimentUtil = ({ experiment }) => {
+  const lastRunStarted = new Date(experiment.last_run_created_at);
```

Comment:
if last_run_created_at is not a date we should just show `-`

for the cases where the experiment is new and there is no latest